app/models/application_choice.rb

```diff
@@ -1,4 +1,5 @@
 # Candidates can apply to multiple courses via a single Application Form.
 class ApplicationChoice < ApplicationRecord
   belongs_to :application_form, touch: true
+  enum status: { application_complete: 0, conditional_offer: 1 }
```

Comment:
I'm slightly paranoid about `enum`, because we'd have these integers in the database that don't mean anything without the right version of the code.

I *think* the column can be a string and we can do:

```
{ application_complete: "application_complete", conditional_offer: "conditional_offer" }
```

Would that be better?